### ISSUE TITLE
Issue with PageList class

### ISSUE THREAD

**Issue by Eein** [rainlab/pages-plugin#37]
I was attempting to use the findByUrl method in the PageList class and it squawked about RouterHelper not being loaded.

```Class 'RainLab\Pages\Classes\RouterHelper' not found```

When I resolve that dependency in PageList.php I get...

```Class 'RainLab\Pages\Classes\Config' not found```

When I resolve that, I get...

```Call to undefined method RainLab\Pages\Classes\PageList::getRouterObject()```

With a code dump of: (emphasis on $router = $this->getRouterObject(); )

```
/*
             * Find the page by URL and cache the route
             */
  
            if (!$fileName) {
               $router = $this->getRouterObject();
  
                if ($router->match($url)) {
                    $this->parameters = $router->getParameters();
  
                    $fileName = $router->matchedRoute();

```

I'm guessing this is a fragment from laravel 4, and I can get the data I want by other means, but I figured I'd post this here. :)

**Comment by daftspunk** [comment]
Fixed by #36

**Comment by daftspunk** [comment]
I think this method is not supposed to be here, try using `RainLab\Pages\Classes\Router::findByUrl` instead.
